data/scenarios/Challenges/_flower-count/judge.sw

```diff
@@ -0,0 +1,35 @@
+def win =
+  try {
+    firestarter <- robotNamed "firestarter";
+    halt firestarter
+  } {};
+  try {
+    fire <- robotNamed "fire";
+    halt fire; wait 1;
+    reprogram fire { selfdestruct };
+  } {};
+  create "gold"
+end
+
+def judgeCount : Int -> Cmd Unit = \actual.
+  watch down;
+  wait 1024;
+  s <- scan down;
+  case s
+    (\_. return ())
+    (\p.
+      try {
+        let c = (read p : Int) in
+        if (c == actual) { win } {}
+      } {}
+    )
+end
+
+def forever = \c. c; forever c end
+
+def judge =
+  numFlowers <- resonate "flower" ((-59,-19),(60,20));
```

Comment:
I had a (off-by-one) bug in my code that I finally diagnosed by entering `creative` mode to use my own `resonate` command.  Perhaps that is the intended use of `creative` mode, but it may be fun to have an "in-scenario" method to double-check one's count.  Perhaps some oblique way to obtain a [`gradiator`](https://github.com/swarm-game/swarm/blob/83350513c7a4e16df591fac7da601be658baa0d5/data/scenarios/Challenges/Sliding%20Puzzles/3x3.yaml#L156) that is an instant-lose condition, but nonetheless can help a player debug.